src/ica_api.c

```diff
@@ -1126,7 +1130,11 @@ unsigned int ica_rsa_crt(ica_adapter_handle_t adapter_handle,
 		rc = ica_fallbacks_enabled ?
 			rsa_crt_sw(&rb) : ENODEV;
 	else {
-		rc = ioctl(adapter_handle, ICARSACRT, &rb);
+		if (any_card_online)
+			rc = ioctl(adapter_handle, ICARSACRT, &rb);
+		else
+			rc = 1;
```

Comment:
same here

Reply:
I agree, ENODEV is better here. Code changed.